crypto/rand/randfile.c

```diff
@@ -104,6 +104,12 @@ static __FILE_ptr32 (*const vms_fopen)(const char *, const char *, ...) =
 
 #define RFILE ".rnd"
 
+#ifdef EINTR
+# define INTERRUPTED(in) (ferror(in) && errno == EINTR)
```

Comment:
I don't see that it says anywhere that `errno` is actually meaningful at this point and concern is that suggestion is based on *assumption* or de-facto behaviour on OS of popular choice, rather than *standardized* behaviour. What I'm saying is that this fact has to be at least recognized and discussed. In the context it's probably appropriate to elaborate on this in commentary...